### PR TITLE
Mega arachnid

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It is a giant angry spider. It lurks semi transparent, and when aggrod will attempt to shoot its prey, applying a leg trap to them.

If their target is leg trapped or below 50 health, they will go melee them.

If their target isn't leg trapped or below 50 health, but their ranged attack is off cooldown, they will attempt to go shoot and snare them.

If their target is not snared and their ranged attack isn't ready, they will become semi transparent again and attempt to flee. 

Someone on the forums suggested "flesh hooks" and I don't remember who but that was the start of this idea so thank you.

@WJohn sprited this

### CODE REVIEW

**Comment by WJohn** [comment]
i'm so unhappy  with the front/back sprites but w/e i sprited this thing in a couple of days on a whim anyway so it's not like i put effort into it or expected it to be used or anything

**Comment by Armhulen** [comment]
Can it look thematic to all the other spiders kthx

**Comment by XDTM** [comment]
spiders are arachnics but arachnids don't have to be spiders

**Comment by FantasticFwoosh** [comment]
This a 64x64 sprite or a REALLY fat spider? Correct @Cyberboss it could well be a giant scorpion instead.

File: code/modules/mob/living/simple_animal/hostile/jungle_mobs.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	alpha = 50
+
+/obj/item/projectile/mega_arachnid
+	name = "flesh snare"
+	nodamage = 1
+	damage = 0
+	icon_state = "tentacle_end"
+
+/obj/item/projectile/mega_arachnid/on_hit(atom/target, blocked = 0)
+	if(iscarbon(target) && blocked < 100)
```
Comment (kevinz000) [inline code/modules/mob/living/simple_animal/hostile/jungle_mobs.dm]: good it respects RTA

**Comment by Repukan** [comment]
Can we harvest the snares for silk tho?

**Comment by FantasticFwoosh** [comment]
We should be able to harvest any web object for silk methinks. Least give a use for spiders.

**Comment by Shadowlight213** [comment]
Looks more like a scorpion or a really big face hugger than a spider.
I wanted to make a giant mechanical spider joke, but it's ruined now.

**Comment by ChangelingRain** [comment]
This should really have a different sprite, the palette won't mesh well with the aboveground plants and such.

**Comment by Armhulen** [comment]
I agree, embrace the spiders Kor.

**Comment by KorPhaeron** [comment]
I don't have different sprites at this time

**Comment by lzimann** [comment]
someone can sprite in the future

**Pull request merged**